sdk/storage/azure-storage-queue/azure/storage/queue/_message_encoding.py

```diff
@@ -137,7 +146,7 @@ def decode(self, content, response):
 class NoEncodePolicy(MessageEncodePolicy):
     """Bypass any message content encoding."""
 
-    def encode(self, content):
+    def encode(self, content: object) -> object:
         if isinstance(content, bytes) and sys.version_info > (3,):
```

Comment:
Lets get rid of the version check
Lets make it `str`
Since the top-level returns `str` and this doesn't do any transforms, lets make it `str` only